src/main/java/medicaldoctor/controllers/LookUp.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor.controllers;
+
+//LookUp table for all FXML files
+public class LookUp {
+    public static final String LOGIN_SCREEN_NAME = "Login";
+    public static final String LOGIN_SCREEN = "Login.fxml";
```

Comment:
even if we did have to have two keys, you can define `X` in terms of `X_NAME`

Reply:
what do you mean? like LOGIN_SCREEN + ".fxml" ?

Reply:
yes.

but once again, only one unique key is needed.